packages/lesswrong/components/languageModels/AutocompleteModelSettings.tsx

```diff
@@ -0,0 +1,503 @@
+          {expanded ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+          <h4>{author.name}</h4>
+        </div>
+        <div className={classes.sliderContainer}>
+          <div className={classes.tokenCount}>
+            <span>Posts Tokens:</span>
+            <span>{Math.round(postsTokens)}</span>
+          </div>
+          <input
+            type="range"
+            min="0"
+            max={MAX_AUTHOR_TOKENS}
+            value={postsTokens}
+            onChange={(e) => handleTokenChange("posts", Number(e.target.value))}
+          />
+        </div>
+        <div className={classes.sliderContainer}>
+          <div className={classes.tokenCount}>
+            <span>Comments Tokens:</span>
+            <span>{Math.round(commentsTokens)}</span>
+          </div>
+          <input
+            type="range"
+            min="0"
+            max={MAX_AUTHOR_TOKENS}
+            value={commentsTokens}
+            onChange={(e) => handleTokenChange("comments", Number(e.target.value))}
+          />
+        </div>
+      </div>
+      {expanded && (
+        <div className={classes.authorContent}>
+          <SelectableList
+            items={authorContent[author.userId]?.posts || []}
+            selectedItems={selectedItems}
+            onToggle={onToggle}
+            onSelectAll={onSelectAll}
+            ItemComponent={({ item }) => <Components.PostsItem post={item as PostsListWithVotes} />}
+            classes={classes}
+          />
+          <SelectableList
+            items={authorContent[author.userId]?.comments || []}
+            selectedItems={selectedItems}
+            onToggle={onToggle}
+            onSelectAll={onSelectAll}
+            ItemComponent={({ item }) => (
+              <Components.CommentsNode
+                treeOptions={{ forceSingleLine: true } as CommentTreeOptions}
+                comment={item as CommentsList}
+              />
+            )}
+            classes={classes}
+          />
+        </div>
+      )}
+    </div>
+  );
+};
+
+const debouncedSaveSelection = debounce((selectedItems, posts, comments, authorContent) => {
+  const selectedIds = Object.entries(selectedItems)
+    .filter(([, isSelected]) => isSelected)
+    .map(([id]) => id);
+
+  const selectedPosts = selectedIds.filter(
+    (id) =>
+      posts?.some((post: PostsListWithVotes) => post._id === id) ||
+      featuredAuthors.some((author) =>
+        authorContent[author.userId]?.posts?.some((post: PostsListWithVotes) => post._id === id),
+      ),
+  );
+  const selectedComments = selectedIds.filter(
+    (id) =>
+      comments?.some((comment: CommentsList) => comment._id === id) ||
+      featuredAuthors.some((author) =>
+        authorContent[author.userId]?.comments?.some((comment: CommentsList) => comment._id === id),
+      ),
+  );
+
+  localStorage.setItem("selectedTrainingPosts", JSON.stringify(selectedPosts));
+  localStorage.setItem("selectedTrainingComments", JSON.stringify(selectedComments));
```

Comment:
Just FYI, domains are limited to something like 5mb of content saved in localstorage, which is not actually a whole lot.